tests/test_transactions.py

```diff
@@ -32,6 +32,7 @@ class TestTransactions(test.TruncationTestCase):
         saved_event = await Tournament.filter(name="Updated name").first()
         self.assertIsNone(saved_event)
 
+    @test.skip("logically flawwed")
     async def test_nested_transactions(self):
```

Comment:
We have determined that nested transactions are not done right, but would still like this to run...

Reply:
then the test needs to be reworked since it reverts the thing it is checking upon failure atm